src/operator/cross_device_copy.cc

```diff
@@ -60,6 +60,16 @@ class CrossDeviceCopyProp : public OperatorProperty {
     return true;
   }
 
+  bool InferType(std::vector<int> *in_type,
```

Comment:
How would this fix CI?

Reply:
Looks like I pulled an additional commit while rebasing, seems obvious.